README.md

```diff
@@ -199,7 +199,7 @@ Instead of passing the absolute path to this repo, you may instead use
 
 #### Deployment including nginx
```

Comment:
Do we still need the deployment without nginx in the Readme? Its not really useful to have the app without the static files, right?

Reply:
I had the same thoughts but wasn't sure if that intermediate step mught be helpful to underdtand the architecture of the containerized app..?
If you think that's not the case, I'll gladly remove it!

Reply:
volumes and settings are in the docker-compose.yaml The ports for the Django app are not directly there but you wrote in the readme that nginx does the forwarding so one can find that as well. And the secret is imported from the settings.
I guess its fine. We can just add parts back in in case we need them.